oefen_api/text-api.py

```diff
@@ -3,13 +3,17 @@
 app = Flask(__name__)
 
 @app.route("/")
-def schrijf_json():
+def json_formatting():
+    data = None
     with open("text.json") as myfile:
-        data = myfile.read()
-
+        try:
+            data = myfile.read()
```

Comment:
De json lib heeft een [functie](https://docs.python.org/3/library/json.html#json.load) om direct uit een file pointer te lezen (wat je nu in `myfile` stopt)
Dan heb je de latere `json.loads` niet meer nodig.